middleware/heartbeat.go

```diff
@@ -10,12 +10,20 @@ import (
 // can make a request before hitting any routes. It's also convenient
 // to place this above ACL middlewares as well.
 func Heartbeat(endpoint string) func(http.Handler) http.Handler {
+	return HeartbeatFunc(endpoint, func() []byte {
+		return []byte(".")
+	})
+}
+
+// HeartbeatFunc is similar to Heartbeat but dynamically allows for a dynamic
+// payload to be returned
+func HeartbeatFunc(endpoint string, payload func() []byte) func(http.Handler) http.Handler {
```

Comment:
why `func() []byte`?

Reply:
The caller can create any closure that simply returns the payload that they want. This can be dynamic or static as the user sees fit.

We could modify a local copy for our need but if possible evolving the shared repository of middlewares is good imho.